src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(a, b) {
```

Comment:
Here and for other operations as well you can use just one argument to pass to `add()` (for example "number") and then just reuse your result `this.result += number`